lib/hammer_cli_foreman_tasks/task.rb

```diff
@@ -21,21 +34,46 @@ def execute
     end
 
     class ListCommand < HammerCLIForeman::ListCommand
+      extend WithoutNameOption
+      extend ActionField
+
       output do
         field :id, _('ID')
         field :action, _('Name')
-        field :username, _('Owner')
+        field :state, _('State')
+        field :result, _('Result')
         field :started_at, _('Started at'), Fields::Date
         field :ended_at, _('Ended at'), Fields::Date
+        field :username, _('Owner')
+
+        from :humanized do
+          field :errors, _('Task errors'), Fields::List, :hide_blank => true
+        end
+      end
+
+      build_options
+    end
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
+      extend WithoutNameOption
+      extend ActionField
+
+      output do
+        field :id, _('ID')
+        field :action, _('Name')
```

Comment:
similarly as in listing, lets call it `action`. Also, please add `label` to the info, as it's also useful in details.

Reply:
what's the difference between label and action?